### PR TITLE
Documented new 2 step model registration

### PR DESCRIPTION
**Pull request by validbeck**
## Internal Notes for Reviewers

> For sc-5499.

### Working with model workflows
- Added a new `smallcaps` style to our stylesheet and updated the style guide
- Applied it to our relevant "MODEL STATUS" UI text as that is how it appears in the UI
- This is easily applied via the `[model status]{.smallcaps}` format

| Example usage | Style guide update |
|---|---|
|Screenshot 2024-07-17 at 9 07 52 AM|Screenshot 2024-07-17 at 9 29 18 AM|

> I also preemptively updated some of the casing I made more unified in this PR over in `frontend`: https://github.com/validmind/frontend/pull/870

### CODE REVIEW

File: site/guide/model-inventory/register-models-in-inventory.qmd [hunk 1]
```diff
@@ -17,19 +17,28 @@ To be able to document models using ValidMind's model documentation and validati
...
-   - Select the relevant model methodology being used
-   - Select the relevant model use case
-   - Provide a purpose statement to explain what the model will be used for
-   - Select the preliminary risk tier for the model
-   - If the model is a vendor model, toggle **Is Vendor Model**[^2] and provide the vendor name.
+   - **Step 1: Basic Model Information** — If no custom fields are defined for your models, simply provide the following information then move on to step #5 below: 
 
-5. Click **Register new model** to create a new entry in the model inventory.
+      - Model name
+      - Type of model, new[^3] or existing
```
Comment (noosheenv) [inline site/guide/model-inventory/register-models-in-inventory.qmd:27]: When I click on **New**, a field to enter the Template pops up:
image

When I click on **Existing**, a field to enter the Status pops up:
image

File: site/guide/model-inventory/register-models-in-inventory.qmd [hunk 2]
```diff
@@ -17,19 +17,28 @@ To be able to document models using ValidMind's model documentation and validati
...
+      - Associated business unit
+      - Use case
+      - If the model is a vendor model, toggle **Is Vendor Model**[^4] and provide the vendor name
+      - Purpose: Explain what the model will be used for
+      - Preliminary risk tier
```
Comment (noosheenv) [inline site/guide/model-inventory/register-models-in-inventory.qmd:32]: The owners field can also be modified:
image

File: site/guide/model-inventory/register-models-in-inventory.qmd [hunk 3]
```diff
@@ -17,19 +17,28 @@ To be able to document models using ValidMind's model documentation and validati
...
-5. Click **Register new model** to create a new entry in the model inventory.
+      - Model name
+      - Type of model, new[^3] or existing
+      - Associated business unit
+      - Use case
+      - If the model is a vendor model, toggle **Is Vendor Model**[^4] and provide the vendor name
+      - Purpose: Explain what the model will be used for
+      - Preliminary risk tier 
+
+   - **Step 2: Additional Model Information** — If custom fields are defined for your organization:
```
Comment (noosheenv) [inline site/guide/model-inventory/register-models-in-inventory.qmd:34]: Custom fields were defined for my organization, however this step didn't apply. 
HOWEVER I created a "Required on registration" custom field and then this step applied.

**Review by noosheenv — CHANGES_REQUESTED** [review]
Just some stuff from the first pass-through of testing these docs. Plus, there is that owners bug in the frontend so it's kinda fun and quirky for now. Good stuff otherwise!

File: site/guide/model-inventory/register-models-in-inventory.qmd (hunk 1 above)
Comment (validbeck) [reply site/guide/model-inventory/register-models-in-inventory.qmd:27]: This is documented in the footnote ;)

File: site/guide/model-inventory/register-models-in-inventory.qmd (hunk 2 above)
Comment (validbeck) [reply site/guide/model-inventory/register-models-in-inventory.qmd:32]: I can't replicate the bug you mentioned, but it's true we can probably add a line for "Assign other or additional model owners" or something like that.

File: site/guide/model-inventory/register-models-in-inventory.qmd (hunk 3 above)
Comment (validbeck) [reply site/guide/model-inventory/register-models-in-inventory.qmd:34]: Hm, are you saying that the step 2 panel didn't show up at all even though you had custom fields on your organization?

File: site/guide/model-inventory/register-models-in-inventory.qmd (hunk 1 above)
Comment (noosheenv) [reply site/guide/model-inventory/register-models-in-inventory.qmd:27]: So should there be another footnote next to **Existing** that says `[^x]: Existing models require a **Model Status**` or will that get documented somewhere else?

**Comment by validbeck** [comment]
### New updates:

- Split out the basic & additional panels into their own sections
- Expanded on the required default fields

Screenshot 2024-07-17 at 3 07 36 PM

**Review by noosheenv — APPROVED** [review]
LGTM!!

**Pull request merged**